mono/mini/exceptions-amd64.c

```diff
@@ -540,7 +540,10 @@ mono_arch_find_jit_info (MonoDomain *domain, MonoJitTlsData *jit_tls,
 		guint8 *unwind_info;
 		guint8 *epilog = NULL;
 
-		frame->type = FRAME_TYPE_MANAGED;
+		if (ji->is_trampoline)
```

Comment:
This sequence of code looks repeated a couple of times. Maybe we should factor it into a common function.